libs/service-portal/graphql/src/hooks/profile/useIslykillSettings.ts

```diff
@@ -2,9 +2,14 @@ import { useQuery } from '@apollo/client'
 import { Query } from '@island.is/api/schema'
 import { GET_ISLYKILL_SETTINGS } from '../../lib/queries/getIslykillSettings'
 
-export const useIslykillSettings = () => {
+interface Settings {
+  skip: boolean
+}
+
+export const useIslykillSettings = (settings?: Settings) => {
```

Comment:
Is this hook, or the `settings` argument used? Since you're using the codegen hook.

Reply:
Absolutely. Removing this completely.